verifier/summary_template.html

```diff
@@ -14,15 +14,22 @@
 
       // The summary data for all the tests.
       let exec_summary_json;
-      fetch('exec_summary.json')
-        .then((response) => response.json())
-        .then((data) => {exec_summary_json = data});
+      Promise.all([
+          fetch_promise = fetch('exec_summary.json')
+              .then((response) => response.json())
+              .then((data) => {exec_summary_json = data}),
+          new Promise((resolve, reject) => {
+              $(document).ready(resolve);
+          })
+      ]).then(([fetch_promise, _ready]) => {
+          google.charts.setOnLoadCallback(drawChart);
+      });
 
       // Load the Visualization API and the corechart package.
       google.charts.load('current', {'packages':['corechart']});
 
       // Set a callback to run when the Google Visualization API is loaded.
-      google.charts.setOnLoadCallback(drawChart);
+      // google.charts.setOnLoadCallback(drawChart);
```

Comment:
Nit: You don't need this block of code any more; if you'd like, move the comment up into the new `.then()` callback on the Promise.all